joinslack.html

```diff
@@ -1,6 +0,0 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <meta http-equiv="Refresh" content="0; url=//join.slack.com/t/technica-2019/shared_invite/enQtODA4NDA3MzE2OTc5LWIxODk2NDI1MmRhYWJlNDZhNTNiNjQyNGRmNWQ1NjAwOGRhYzY0MGY0NDhiMTkzOGZiZjI3Mzk4OGM5NTZiZjY" />
```

Comment:
Can we keep this? Later years might need it

Reply:
this is the invite url for the 2019 event slack not the organizer slack that we use